### DISCUSSION TITLE
Outlook not accepting ical Objects and rendering them as calendar invites

### DISCUSSION THREAD

**Discussion by nishthaGupta** [ical4j/ical4j#784] [General]
This is the latest observation. Just a week back, this ical object, was working:

BEGIN:VCALENDAR
PRODID:-//Events Calendar//iCal4j 1.0//EN
VERSION:2.0
CALSCALE:GREGORIAN
METHOD:REQUEST
BEGIN:VEVENT
DTSTAMP:20180508T085106Z
DTSTART:20180526T020000
DTEND:20180526T030000
SUMMARY:eventName
TZID:Asia/Kolkata
ATTENDEE;ROLE=REQ-PARTICIPANT;RSVP=FALSE:MAILTO:nishthagupta25@gmail.com
ORGANIZER:MAILTO:nishthagupta65@gmail.com
LOCATION:XYZ
SEQUENCE:1056076717
UID:abcd
DESCRIPTION:Desc
END:VEVENT
BEGIN:VTIMEZONE
TZID:Asia/Kolkata
TZURL:http://tzurl.org/zoneinfo-outlook/Asia/Kolkata
X-LIC-LOCATION:Asia/Kolkata
BEGIN:STANDARD
TZOFFSETFROM:+0530
TZOFFSETTO:+0530
TZNAME:IST
DTSTART:19700101T000000
END:STANDARD
END:VTIMEZONE
END:VCALENDAR

Now, this doesn't work on outlook client at all. This ical object is not getting rendered as calendar invite on outlook. **Is there a new advancement from your side to support Outlook client?** 

PS: This still works for Gmail client.

Urgent issue.

**Comment by Attitood** [comment]
This may not be an issue with ical4j.  I had this issue & resolved it doing this:
Content-Type: text/calendar
to
Content-Type: text/calendar; charset="UTF-8"; method=REQUEST

**Comment by prasannajeetg** [comment]
I am still facing this issue with the below-mentioned calendar invite and it is not working despite of using the solution mentioned by @Attitood  

BEGIN:VCALENDAR
PRODID:-//XXX//iCal4j 2.1//EN
VERSION:2.0
CALSCALE:GREGORIAN
METHOD:REQUEST
BEGIN:VTIMEZONE
TZID:Etc/GMT
TZURL:http://tzurl.org/zoneinfo/Etc/GMT
X-LIC-LOCATION:Etc/GMT
BEGIN:STANDARD
TZOFFSETFROM:+0000
TZOFFSETTO:+0000
TZNAME:GMT
DTSTART:16010101T000000
RDATE:16010101T000000
END:STANDARD
END:VTIMEZONE
BEGIN:VEVENT
DTSTAMP:20180816T090215Z
TZID:Etc/GMT
STATUS:CONFIRMED
ORGANIZER:mailto:donotreply@xxxx.com
UID:9bdfd482-55cc-4c29-9912-e1ad653333dd
DTSTART:20180820T153000Z
DTEND:20180820T163000Z
LOCATION:XXXXXXXX
SUMMARY:Test Summary
DESCRIPTION:Test Description
LAST-MODIFIED:20180816T090215Z
SEQUENCE:0
ATTENDEE;CN=XXXX:mailto:xxx@xxxxx.com
ATTENDEE;CN=XXXX:mailto:xxxx@xxxx.com
END:VEVENT
END:VCALENDAR

Though this doesn't seem to be a problem with ical4j but is there any outlook update or improvement which might be causing this?
